internal/cloud_provider/resource_cloud_provider.go

```diff
@@ -421,8 +441,7 @@ func findProvider(providers []client.Provider, uuid string) (*client.Provider, e
 	return nil, fmt.Errorf("could not find provider %s", uuid)
 }
 
-func resourceCloudProviderRead(ctx context.Context, d *schema.ResourceData, meta interface{}) (
-	diag.Diagnostics) {
+func resourceCloudProviderRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
```

Comment:
Similar change to keep within lint limit

Reply:
Humm I am using the official terraform golang formating, so not sure the lint limit make sense here